src/app/visit/visit.component.ts

```diff
@@ -290,9 +290,9 @@ export class VisitComponent implements OnInit, AfterViewInit, OnDestroy {
                 {
                     table: {
                         widths: ['50%', '50%'],
-                        body: this.visitPoints.map((point) =>
-                            this.getPdfRow(point._id),
-                        ),
+                        body: this.visitPoints
+                            .filter((vt) => vt.code.indexOf('P', 0))
```

Comment:
![image](https://github.com/Johngtka/BioMagnetic-app/assets/11686968/e56d8318-301b-4fc7-a353-8ccf4c7d5461)

this is not working correctly, what does the indexOf returns?

Reply:
index of is returns a number

Reply:
so? what number returns when character is not found? you filter logic should check if the 'P' is found or not? Current filter removes all objects that have a code starting with 'P'. When you do changes please check if it works first, set a debugger and analyse the code if it works correctly.

Reply:
any progress??

Reply:
any update?

Reply:
On this moment no, but i prepare to recheck all of changes in this pull request in a few days